src/bindgen/library.rs

```diff
@@ -708,6 +711,42 @@ impl Library {
         };
         result.items.sort_by(ordering);
 
+        // Collect all possible member fucntions
+        if self.config.language == Language::Cxx && self.config.structure.generate_member_functions {
+            let items = ::std::mem::replace(&mut result.items, Vec::new());
+            for item in items {
+                match item {
+                    PathValue::Struct(mut s) => {
+                        let ty = Type::Path(s.name.clone(), Vec::new());
+                        if let Some(functions) = oop.remove(&ty) {
+                            let opaque = OpaqueItem {
+                                name:s.name.clone(),
+                                generic_params: s.generic_params.clone(),
+                                annotations: s.annotations.clone(),
+                                documentation: s.documentation.clone(),
+                            };
+                            result.items.push(PathValue::OpaqueItem(opaque));
+                            s.functions = functions.into_iter()
```

Comment:
Can you add a comment mentioning essentially that you're stripping out the `this` argument?